hubspot/connection/__init__.py

```diff
@@ -228,11 +230,11 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._session.close()
 
 
-_AuthenticationKey = Record.create_type('_AuthenticationKey', 'key_value')
+_AuthenticationKey = Record.create_type(str('_AuthenticationKey'), str('key_value'))
```

Comment:
I'm not sure why, but this time after removing those I am not getting the same errors.  So, that's fantastic, I think (except not knowing *why* the behavior changed).